CMakeLists.txt

```diff
@@ -56,6 +58,11 @@ option(USE_GOLD_LD "Use gold linker intead of GNU linker" ON)
 option(USE_CCACHE "Turn on ccache usage" ON)
 option(USE_DISTCC "Turn on distributed build_usage" OFF)
 
+if(UNIX AND QNXNTO)
+  set(BUILD_BT_SUPPORT OFF)
```

Comment:
@BSolonenko can we make BUILD_BT_SUPPORT and BUILD_USB_SUPPORT as options?

Reply:
@AKalinich-Luxoft Fixed in b2b51c312fc57007af9a6626c0e88bfe2f0b594e